src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -196,6 +197,18 @@ public Task UserRemoveFromAllGroupsAsync(string userId, CancellationToken cancel
             return _restClient.SendAsync(api, HttpMethod.Delete, _productInfo, cancellationToken: cancellationToken);
         }
 
+        public async Task<bool> IsUserInGroup(string userId, string groupName, CancellationToken cancellationToken = default)
+        {
+            var isUserInGroup = false;
+            var api = _restApiProvider.GetUserGroupManagementEndpoint(_appName, _hubName, userId, groupName);
+            await _restClient.SendAsync(api, HttpMethod.Get, _productInfo, handleResponse: (request, response) =>
+                {
+                    isUserInGroup = response.StatusCode == HttpStatusCode.OK;
```

Comment:
what about 503?

Reply:
when service is unavailable?

Reply:
Let's throw when the status code isn't defined in the [doc.](https://github.com/Azure/azure-signalr/blob/dev/docs/rest-api.md#Check-user-existence-in-a-group)